docs/toc.yml

```
@@ -1156,12 +1153,19 @@
 
 - name: Patterns and practices
   items:
-  - name: Share code from VSTO Add-in to Office add-in
-    href: tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md
   - name: Batching custom function remote calls
     href: excel/custom-functions-batching.md
     displayName: Excel, Custom Functions
   - name: Authentication
     href: develop/authentication-pnp.md
   - name: Handle high DPI and DPI scaling in your Office solution
     href: /office/client-developer/ddpi/handle-high-dpi-and-dpi-scaling-in-your-office-solution
+
+- name: VSTO transition resources
```

If this includes COM resources, should it say so? e.g.:
```suggestion
- name: VSTO and COM transition resources
```